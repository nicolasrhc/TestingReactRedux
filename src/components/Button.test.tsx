import React from "react";

describe("Button Test", () => {
    it("Debe renderizarse correctamente", () => {
        expect(1).toBe(1);
    });
});
