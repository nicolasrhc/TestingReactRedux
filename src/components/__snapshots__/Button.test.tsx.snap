// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Test Debe renderizarse correctamente 1`] = `
<button
  disabled={false}
  onClick={[Function]}
/>
`;
